### PR TITLE
Ed alembic

### PR DESCRIPTION
**Pull request by l-vincent-l**
Change database creation of ed.
We now use alembic instead of sql scripts to better handle migrations

### CODE REVIEW

File: source/sql/README [hunk 1]
```diff
@@ -0,0 +1,26 @@
+Initialization
+--------------
+
+You need a postgres/postgis database.
+You have to change the variable sqlalchemy.url in alembic.ini
+If your database is new, you have to run:
+    PYTHONPATH=. alembic upgrade head
+Else, you need to initialize alembic on your database with:
+    PYTHONPATH=. alembic stamp head
```
Comment (kinnou02) [inline source/sql/README:9]: Only the first thime

File: source/sql/alembic/versions/35c20e6103d6_create_tables.py [hunk 2]
```diff
@@ -0,0 +1,632 @@
...
+    sa.Column('school_vehicle', sa.BOOLEAN(), nullable=False),
+    sa.PrimaryKeyConstraint('id'),
+    schema='navitia'
+    )
+    op.create_table('parameters',
+    sa.Column('beginning_date', sa.DATE(), nullable=True),
+    sa.Column('end_date', sa.DATE(), nullable=True),
+    sa.Column('timezone', sa.TEXT(), nullable=True),
+    sa.Column('shape', ga.Geography(geometry_type='POLYGON', srid=4326, spatial_index=False), nullable=True),
+    sa.Column('shape_computed', sa.BOOLEAN(), nullable=True),
```
Comment (kinnou02) [inline source/sql/alembic/versions/35c20e6103d6_create_tables.py]: actually we have a default value for this attribute

File: source/sql/alembic/versions/35c20e6103d6_create_tables.py [hunk 3]
```diff
@@ -0,0 +1,632 @@
...
+    schema='georef'
+    )
+    op.create_table('ticket',
+    sa.Column('ticket_key', sa.TEXT(), nullable=False),
+    sa.Column('ticket_title', sa.TEXT(), nullable=True),
+    sa.Column('ticket_comment', sa.TEXT(), nullable=True),
+    sa.PrimaryKeyConstraint('ticket_key'),
+    schema='navitia'
+    )
+    op.create_table('administrative_regions',
```
Comment (kinnou02) [inline source/sql/alembic/versions/35c20e6103d6_create_tables.py]: I don't have this table currently

File: source/sql/alembic/versions/35c20e6103d6_create_tables.py [hunk 4]
```diff
@@ -0,0 +1,632 @@
...
+    sa.PrimaryKeyConstraint('id'),
+    schema='navitia'
+    )
+    op.create_table('message_status',
+    sa.Column('id', sa.INTEGER(), nullable=False),
+    sa.Column('name', sa.TEXT(), nullable=False),
+    sa.PrimaryKeyConstraint('id'),
+    schema='realtime'
+    )
+    op.create_table('alembic_version',
```
Comment (kinnou02) [inline source/sql/alembic/versions/35c20e6103d6_create_tables.py]: ?

File: source/sql/alembic/versions/35c20e6103d6_create_tables.py [hunk 5]
```diff
@@ -0,0 +1,632 @@
...
+    op.create_table('odt_type',
+    sa.Column('id', sa.BIGINT(), nullable=False),
+    sa.Column('name', sa.TEXT(), nullable=False),
+    sa.PrimaryKeyConstraint('id'),
+    schema='navitia'
+    )
+    op.create_table('origin_destination',
+    sa.Column('id', sa.BIGINT(), nullable=False),
+    sa.Column('origin_id', sa.TEXT(), nullable=False),
+    sa.Column('origin_mode', postgresql.ENUM(u'Zone', u'StopArea', u'Mode', name='fare_od_mode'), nullable=False),
```
Comment (kinnou02) [inline source/sql/alembic/versions/35c20e6103d6_create_tables.py]: it would be easier to declare the enum only one time

**Comment by antoine-de** [comment]
Jenkins, test this please

**Pull request merged**